packages/subgraph/tests/dao/acl.test.ts

```diff
@@ -102,7 +118,32 @@ test('Run dao (handleGranted) mappings with mock event', () => {
     daoPackageEntityID
   );
 
+  // packages
+  if (supportsMajorityVoting) {
```

Comment:
With an ERC20 voting interface, we can change this logic to `supportsERC20Voting`